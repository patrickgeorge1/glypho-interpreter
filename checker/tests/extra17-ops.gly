WWW4HHHR444'9,,eyKyGi7v7QJJJ