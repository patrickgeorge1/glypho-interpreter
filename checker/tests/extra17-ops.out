279463716208988557014
